enterprise/frontend/src/metabase-enterprise/whitelabel/components/LandingPageWidget/LandingPageWidget.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from "react";
+import { t } from "ttag";
+import { Text } from "metabase/ui";
+import type { EnterpriseSettings } from "metabase-enterprise/settings/types";
+import { SettingInputBlurChange } from "./LandingPageWidget.styled";
+
+interface Props {
+  settingValues: EnterpriseSettings;
+  onChange: (value: string) => void;
+  onChangeSetting: (
+    key: "landing-page",
+    value: EnterpriseSettings["landing-page"],
+  ) => Promise<void>;
+}
+
+export const LandingPageWidget = ({
+  onChangeSetting,
+  settingValues,
+}: Props) => {
+  const [error, setError] = useState<string | null>(null);
+
+  const normalize = (value: string | number | null) => {
+    if (typeof value !== "string") {
+      return value;
+    }
+    const normalizedValue = value.trim();
+    return normalizedValue === "" ? null : normalizedValue;
+  };
+
+  const handleChange = async (rawValue: string) => {
+    const value = rawValue.trim();
+    const url = new URL(value, location.origin);
```

Comment:
check if `isMetabaseUrl` does what you need here.

not entirely sure if it matters here, but be aware that some people host metabase on a subpath like `https://example.com/metabase/`